jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -177,163 +180,29 @@ public int getMaxEntries()
         return maxEntries;
     }
 
-    /**
-     * @return the default maximum multiplex count of entries
-     * @deprecated Multiplex functionalities will be removed
-     */
-    @ManagedAttribute("The default maximum multiplex count of entries")
-    @Deprecated
-    public int getMaxMultiplex()
-    {
-        return maxMultiplex == -1 ? 1 : maxMultiplex;
-    }
-
     /**
      * <p>Retrieves the max multiplex count for the given pooled object.</p>
      *
      * @param pooled the pooled object
      * @return the max multiplex count for the given pooled object
-     * @deprecated Multiplex functionalities will be removed
      */
-    @Deprecated
     protected int getMaxMultiplex(T pooled)
```

Comment:
Since you added a constructor that takes a `ToIntFunction`, there is no need to keep this method as protected anymore, it can be made private.